eth_crowdfund_be/eth_crowdfund_api/src/tests/test_campaign.py

```diff
@@ -0,0 +1,155 @@
+import pdb
+import unittest 
+import os 
+import json 
+# import flask_sqlalchemy
+from ..models.Campaign import Campaign
+from ..app import create_app, db
+
+def func(x):
+  return x + 1
+# basic test
+def test_answer():
+  assert func(3) == 4
+
+class CampaignTest(unittest.TestCase):
+  # set up the tests with 2 campaigns
+  # This runs before EVERY test. not once before all
+  def setUp(self):
+    self.app = create_app('testing')
+    self.client = self.app.test_client
+    self.campaign1 = {
+        "name": "camp",
+        "description": "desc1",
+        "image": "image.edu",
+        "contributors": 1,
+        "upvote": 2,
+        "manager": "3",
+        "address": "4",
+        "min_contribution": 5.0
+    }
+    self.campaign2 = {
+        "name": "Eth",
+        "description": "desc2",
+        "image": "image.com",
+        "contributors": 100,
+        "upvote": 222,
+        "manager": "13",
+        "address": "14",
+        "min_contribution": 50.2
+    }
+    
+    with self.app.app_context():
+      # create all db objects
+      db.create_all()
+      # delete test campaigns by name
+      existing_campaign1 = Campaign.get_campaign_by_name(self.campaign1["name"])
+      if existing_campaign1:
+        existing_campaign1.delete()
+      existing_campaign2 = Campaign.get_campaign_by_name(self.campaign2["name"])
```

Comment:
Did you create `get_campaign_by_name`? Where is it located?

Reply:
`get_campaign_by_name` is a static method thats created in the Campaign model (`src/models/campaign.py`)